src/components/feedback/feedback.js

```diff
@@ -74,25 +74,30 @@ class Feedback extends React.Component {
     return {
       event: 'Sent docs feedback',
       // set user if available (needed for forward-event request)
-      userId: this.props.userName || undefined,
-      ...(!this.props.userName && { anonymousId: anonymousId }),
+      ...(this.props.user && this.props.user && { userId: this.props.user.id }),
```

Comment:
Is `this.props.user` supposed to be in here twice?

Reply:
Whoops! Good find.